tests/bsim/bluetooth/host/privacy/device/src/test_undirected_peripheral.c

```diff
@@ -232,14 +232,10 @@ static void update_adv_params(struct bt_le_ext_adv *adv, enum adv_param_t adv_pa
 	}
 
 	if (use_ext_adv && adv_params == NCONN_SCAN) {
-		uint8_t data[4] = {0x61, 0x6c, 0x65, 0x64};
-		struct bt_data sd;
+		struct bt_data sd =
+			BT_DATA_BYTES(BT_DATA_NAME_COMPLETE, 'z', 'e', 'p', 'h', 'y', 'r');
```

Comment:
Does that work? 
Or should it be the following? 

```suggestion
			BT_DATA_BYTES(BT_DATA_NAME_COMPLETE, {'z', 'e', 'p', 'h', 'y', 'r'});
```